### PR TITLE
feat: detail of a group

### PR DESCRIPTION
**Pull request by djaiss**
This PR gets the details of a group.

We will do the list of groups in a vault in another PR. This is strictly the detail of a group, accessible from the Group module.

We have a new route `https://chandler.test/vaults/4/groups/1` and therefore a new middleware that makes sure the group is valid in the vault.

@asbiin I'm not sure the `GroupShowViewHelper` method is optimized.

image

### CODE REVIEW

**Comment by djaiss** [comment]
Ping @asbiin

File: domains/Contact/ManageGroups/Web/Controllers/GroupController.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+use App\Vault\ManageVault\Web\ViewHelpers\VaultIndexViewHelper;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Inertia\Inertia;
+
+class GroupController extends Controller
+{
+    public function index(Request $request, int $vaultId)
+    {
+    }
```
Comment (asbiin) [inline domains/Contact/ManageGroups/Web/Controllers/GroupController.php:18]: Is there a reason to keep this empty function?

**Review by asbiin — APPROVED** [review]
Just one question, but apart from that it's great!

**Comment by asbiin** [comment]
And I've optimized `GroupShowViewHelper`

File: domains/Contact/ManageGroups/Web/Controllers/GroupController.php (hunk 1 above)
Comment (djaiss) [reply domains/Contact/ManageGroups/Web/Controllers/GroupController.php:18]: Well, I forgot to remove it :-) But I'll fill this method in the subsequent PR as we'll have to do the index anyway.

**Pull request merged**